src/canvas/__tests__/eventData.test.ts

```diff
@@ -122,3 +122,20 @@ it('A selected subtarget should not fire an event twice', () => {
   } as unknown as TPointerEvent);
   expect(targetSpy).toHaveBeenCalledTimes(1);
 });
+
+it('should fire mouse over/out events on target', () => {
+  const target = new FabricObject({ width: 10, height: 10 });
+  const canvas = new Canvas();
+  canvas.add(target);
+
+  jest.spyOn(target, 'toJSON').mockReturnValue('target');
+
+  const targetSpy = jest.spyOn(target, 'fire');
+  const canvasSpy = jest.spyOn(canvas, 'fire');
+  const enter = new MouseEvent('mousemove', { clientX: 5, clientY: 5 });
+  const exit = new MouseEvent('mousemove', { clientX: 20, clientY: 20 });
+  canvas._onMouseMove(enter);
+  canvas._onMouseMove(exit);
+  expect(targetSpy.mock.calls).toMatchSnapshot();
```

Comment:
@ShaMan123  This could have been `targetSpy.mock.calls.map(([event]) => event)`, then even use `toMatchInlineSnapshot` for a much smaller and nicer snapshot, while being convenient.

Reply:
yes.
stop `toMatchSnapthot`.